src/scenario/scenario.c

```diff
@@ -53,37 +141,41 @@ void scenario_save_state(buffer *buf)
         buffer_write_i16(buf, scenario.invasions[i].attack_type);
     }
 
+    // funds_enemy
```

Comment:
These comments seem redundant as they don't actually explain anything.

Reply:
Removed.
(They corresponded with the variable names of the offsets, but if it wasn't obvious they missed their purpose.)